src/Resources/app/storefront/src/storefront/js/nbbreturnforms.js

```diff
@@ -0,0 +1,38 @@
+const manufacturerSupportFormElementsNumber = 3;
+const productDetailsFormElementsNumber = 5;
+const manufacturerSupportForm = 'manufacturer_support_form';
```

Comment:
Consider to add suffix `ClassName` to where we have to do with vars that hold class names - `manufacturerSupportFormClassName`